### PR TITLE
Ensure everyone uses the less privileged workflow

### LINKED ISSUE #571: Switch to external PR workflow only
Would it be better to have everyone follow the same (less privileged) workflow? This could help us surface issues in things like CI that may not appear until someone outside the organisation try to contribute which would impeded us merging in there changes without some potentially unnecessary back and forth.

_Originally posted by @sedders123 in https://github.com/j5api/j5/pull/539_

We should test the external PR as part of this issue, and update the documenation accordingly.

### PR DESCRIPTION
**Pull request by trickeydan**
This PR partially exists so that I can test the CI

## Checklist

- [x] I have updated the relevant documentation
- [x] I have added the `semver-major`, `semver-minor` or `semver-patch` label as appropriate
- [ ] I would like multiple reviewers to approve my code before merging

## Why do you want to make these changes?

All contributors should use the less-privileged workflow to ensure that our CI behaves appropriately.

## Which parts of the codebase do your changes affect?

None

## How do your changes affect student or volunteer experience?

N/A

## Are your changes related to any existing issues or PRs? How so?

Fixes #571

### CODE REVIEW

**Comment by trickeydan** [comment]
I had to fiddle one setting in CircleCI to fix #460 but this otherwise works.

Once this is merged, I'll mail the list to let them know and we can consider restricting write access a little?

File: docs/development/submitting-changes.rst [hunk 1]
```diff
@@ -27,11 +27,7 @@ Publishing Changes
 
 The first step to submit changes is to publish them so that other contributors can review and give feedback.
 
-The mechanism to publish your changes depends on whether you have write access to the repository. Write access
-is granted by the Steering Council to recurrent, or trusted contributors.
-
-* Contributors with write access should publish a branch on the repository.
-* Contributors without write access should publish as a branch on a fork of the repository.
+Branches should be published on GitHub on a fork of the j5 repository.
```
Comment (sedders123) [inline docs/development/submitting-changes.rst:30]: Nit: Should this be 'to a fork'?
```suggestion
Branches should be published on GitHub to a fork of the j5 repository.
```

Comment (sedders123) [reply docs/development/submitting-changes.rst:30]: Or should it be 'to GitHub on a fork'?

Comment (trickeydan) [reply docs/development/submitting-changes.rst:30]: I prefer your wording, wasn't sure about it when I wrote it.

**Comment by trickeydan** [comment]
Reopening until #609 gets closed and the branch deleted. This is the last branch on the main repo.

After that is merged, we should email the mailing list and restrict writes.

**Pull request merged**